packages/backend/src/clients/github/Github.ts

```diff
@@ -199,6 +199,19 @@ export const createBranch = async ({
         throw new UnexpectedGitError(getErrorMessage(error));
     }
 };
+export const getInstallationToken = async (
```

Comment:
we can use this token in the repo URL, without needing to use octokit during onboarding